reffect/src/trigger/progress/state/state_type.rs

```diff
@@ -0,0 +1,34 @@
+use crate::enums::check_variant_array;
+use const_default::ConstDefault;
+use serde::{Deserialize, Serialize};
+use strum::{AsRefStr, EnumCount, EnumIter, IntoStaticStr, VariantArray};
+
+#[derive(
+    Debug,
+    Clone,
+    PartialEq,
+    AsRefStr,
+    IntoStaticStr,
+    EnumIter,
+    EnumCount,
+    VariantArray,
+    Serialize,
+    Deserialize,
+)]
+pub enum StateType {
+    Available,
```

Comment:
Since available is not functional it should either be removed as option here or somehow inform the user in the UI